virtualmachine/vsphere/vm.go

```diff
@@ -50,7 +50,7 @@ type VMwareLease struct {
 // HTTPNfcLeaseProgress takes a percentage as an int and sets that percentage as
 // the completed percent.
 func (v VMwareLease) HTTPNfcLeaseProgress(p int) {
-	v.Lease.HttpNfcLeaseProgress(v.Ctx, p)
+	v.Lease.HttpNfcLeaseProgress(v.Ctx, int32(p))
```

Comment:
This change should be propagated to callers as well.
i.e.
func (v VMwareLease) HTTPNfcLeaseProgress(p int32) {